routes/votings.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express')
+const Voting = require('../models/Voting')
+
+const router = express.Router()
+
+router.post('/', (request, response, next) => {
+  const { user_id, first_movie_id, second_movie_id } = request.body
+
+  if (user_id === '' || first_movie_id === '' || second_movie_id === '') {
+    const error = { message: 'Information missing.' }
+    return next({ status: 400, message: error.message })
+  }
+
+  const newVote = {
+    user_id,
+    first_movie_id,
+    second_movie_id,
+    isVoted: false,
+    first_movie_won: false,
+  }
```

Comment:
I don't really understand your structure. A new vote is the result of a game that has been played? Or is a new vote a new game that has been started but not ended? Ideally I think the first would be best to save. Otherwise you have to to a `PATCH` request _every time_ someone actually votes on a movie. I think this should be done in the frontend, but I am not 100% sure. 
Just thinking that it would be better to save the end result instead. 🤷‍♀️